dsf-gdb/org.eclipse.cdt.dsf.gdb/src/org/eclipse/cdt/dsf/gdb/launching/FinalLaunchSequence.java

```diff
@@ -652,28 +621,6 @@ public void stepAttachToProcess(final RequestMonitor requestMonitor) {
 		}
 	}
 
-	/**
-	 * If we are dealing with an remote attach session, perform the attach to debugger.
-	 * Bug 528145
-	 * @since 6.6
-	 */
-	@Execute
-	public void stepAttachRemoteToDebugger(final RequestMonitor requestMonitor) {
```

Comment:
This is now API and can't be removed this simply.

Reply:
kept the API with deprecated annotation and its no more called..